domManipulation.js

```diff
@@ -0,0 +1,157 @@
+import { format } from "date-fns";
+
+// Get the current date to convert the forecast for the current date to "Today";
+export function getCurrentDate() {
+  // Convert to format that matches how the result from the network call has been formulated.
+  return format(new Date(), "MMdd");
+}
+
+// Convert the general weather types to corresponding emojis.
+export function weatherIconConverter(weatherType = "") {
+  let weatherEmoji = "";
```

Comment:
Perhaps set a sane default here, and then override it below

Reply:
Set the default to the same emoji as the clear weather type.